src/ecalc/libraries/libecalc/common/libecalc/core/result/results.py

```diff
@@ -45,6 +45,8 @@ class CompressorResult(GenericComponentResult):
     recirculation_loss: TimeSeriesRate
     rate_exceeds_maximum: TimeSeriesBoolean
     outlet_pressure_before_choking: TimeSeriesFloat
+    requested_inlet_pressure: Optional[TimeSeriesFloat]
```

Comment:
Do they need to be Optional? I think suction_pressure and discharge_pressure are required in a compressor energy_usage_model.